src/crystal/system/win32/time.cr

```diff
@@ -53,21 +53,23 @@ module Crystal::System::Time
     ((filetime.dwHighDateTime.to_u64 << 32) | filetime.dwLowDateTime.to_u64).to_f64 / FILETIME_TICKS_PER_SECOND.to_f64
   end
 
-  @@performance_frequency : Int64 = begin
-    ret = LibC.QueryPerformanceFrequency(out frequency)
-    if ret == 0
-      raise RuntimeError.from_winerror("QueryPerformanceFrequency")
-    end
+  @@performance_frequency : Int64?
 
-    frequency
+  private def self.performance_frequency
+    @@performance_frequency ||= begin
+      if LibC.QueryPerformanceFrequency(out frequency) == 0
+        raise RuntimeError.from_winerror("QueryPerformanceFrequency")
+      end
+      frequency
+    end
   end
```

Comment:
I added a `.ticks` method that doesn't raise (not even on overflow) so tracing won't be affected + it directly returns the monotonic time in nanoseconds, which is simpler.

Also the win32 functions are guaranteed to never fail since Windows XP, and we only support Windows 7+ so I removed the error checks.